<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="text_borrar_empleado">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>560</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_telefono_empleado">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_registrar_empleado">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>500</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Introduce email del empleado</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_nombre_empleado">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>95</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>540</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Introduce email del empleado</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellido</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabla_empleados">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>611</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_buscar_empleado">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>560</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>460</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_email_empleado">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_atras">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>610</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="go-previous"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tel√©fono</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_eliminar_empleado">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>600</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_id_empleado">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>460</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_email_buscar_empleado">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>520</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>EMPLEADO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_apellido_empleado">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_actualizar_empleado">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>460</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>Actualizar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_cargo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_mostrar_empleados">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>510</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Mostrar empleados</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_informe">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>610</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 255);
background-color: rgb(170, 0, 127);</string>
    </property>
    <property name="text">
     <string>Informe PDF</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
